app/templates/index.hbs

```diff
@@ -18,6 +18,87 @@
   version becomes the new stable release of Ember.
 </p>
 
+<p>
+  LTS (Long Term Support) releases are done every four minor versions, plus the
+  first and last releases of major versions. An LTS release receives security
```

Comment:
not the first and last, only the last (specifically neither 2.0 nor 3.0 were ever considered for the LTS candidates).